src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2870,4 +2870,10 @@ Activate by holding the key for the character you want to add an accent to, then
   <data name="TextExtractor_Languages.Header" xml:space="preserve">
     <value>Preferred language</value>
   </data>
+  <data name="GeneralPage_HideSystemTrayIcon.Description" xml:space="preserve">
+    <value>You will not be able to access PowerToys settings once you do this.</value>
```

Comment:
When it's said like this, that sounds very... Let's say unlogical.

Reply:
I wrote that as a temporary comment until I could implement the fix to launching PowerToys as a duplicate instance correctly launching settings. I will be pushing that here soon with a more descriptive message.